README.md

```diff
@@ -84,6 +84,13 @@ bin/cert cert-gen --domain <DOMAIN> --cert-type lets_encrypt --email <EMAIL>
 bin/cert enable-https
 ```
 
+Use self generated certificates:
+
+```
+bin/cert cert-install --key <PATH-TO-PRIVKEY> --fullchain <PATH-TO-FULLCHAIN>
+bin/cert enable-https
+```
```

Comment:
Thanks for updating the readme.  Can you also update the help text for the `cert-gen` command.  It currently recommends using a Let's Encrypt certificate and mentions an alternative being self-signed certificates.  That needs changing to also mention use of self-generated certificates.  You'll find the help in `lib/flight_cert/cli.rb`.